### PR TITLE
Missed printing 'D'

### PR DESCRIPTION
**Pull request by saarthdeshpande**
'I' was printed twice and 'D' (distance vector) was not printed. Fixed.

### CODE REVIEW

File: tutorial/cpp/6-HNSW.cpp [hunk 1]
```diff
@@ -55,12 +55,10 @@ int main() {
...
         }
 
-        index.search(nq, xq, k, D, I);
-
-        printf("I=\n");
+        printf("D=\n");
         for (int i = nq - 5; i < nq; i++) {
             for (int j = 0; j < k; j++)
-                printf("%5zd ", I[i * k + j]);
+                printf("%5zd ", D[i * k + j]);
```
Comment (junjieqi) [inline tutorial/cpp/6-HNSW.cpp:61]: Format specifies type 'ssize_t' (aka 'long') but the argument has type 'float' [-Werror,-Wformat]
                printf("%5zd ", D[i * k + j]);
                        ~~~~    ^~~~~~~~~~~~
                        %5f

Comment (junjieqi) [reply tutorial/cpp/6-HNSW.cpp:61]: @saarthdeshpande would you like to fix this part?

Comment (saarthdeshpande) [reply tutorial/cpp/6-HNSW.cpp:61]: Sure

Comment (saarthdeshpande) [reply tutorial/cpp/6-HNSW.cpp]: Done! Added the changes for other tutorial files too.

**Pull request closed without merging**